pkg/metrics/metrics.go

```diff
@@ -103,7 +119,7 @@ The label 'result' can have the following values:
 		prometheus.HistogramOpts{
```

Comment:
Let's also add some buckets to both histograms.

The [DefBuckets](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#pkg-variables) only go up to 10s, which is definitely not enough in a queueing system.

Also, we don't really have resolution below 1s, so maybe we can use something like:

```
[]float64{1, 2.5, 5, 10, 20, 40, 80, 160, 320, 640, 1280, 2560, 5120, 10240}
```

Reply:
Great idea, I saw the cap by 10s already when looking into the metrics for the new scalability tests.